util.py

```diff
@@ -342,21 +354,16 @@ def dict_to_entities(raw_entities, **kwargs):
     return {'entities': entities}
 
 
-def get_dataset_id(dataset_name, **kwargs):
-    datasets = api.list_datasets(**kwargs)
-    for dataset in datasets:
-        if dataset['name'] == dataset_name:
-            return dataset['id']
-
-    return None
+def csv_to_entities(infile, outfile=None, toStdout=False):
+    return dict_to_entities(csv_to_json(infile))
 
 
 def ingest_json(dataset_id, json_file, **kwargs):
-    api.ingest_entities(dataset_id, json.load(open(json_file)), **kwargs)
+    api._ingest_entity_set(dataset_id, dict_to_entities(json.load(open(json_file))), **kwargs)
```

Comment:
prefixing a function with underscore is a protocol for private function that shouldn't be accessible from other libraries. I would recommend that we avoid this approach or rename the function if it is public facing.

or we can move _ingest_entity_set to util file and have util.ingest_csv, util.ingest_json, util.ingest_samples live next to each other and be used by other libraries

I like the second approach better.  But also have concerns about second approach and I feel like those three methods should live in their own class referenced by other libraries.  When I think of utilities class, I view it as an independent set of methods that either format things, calculate things or transform things.  Having util depend on api library is questionable. Feels like they belong in their own library, `api_utils` maybe?

Reply:
I did this to communicate that the function is private and should not be used outside the the module and to keep it from appearing in sphinx documentation.

Then I immediately broke that guideline by using it outside the module.  It seems like the right thing to do until we can get through all the callers of the old `ingest_entities`, now `_ingest_entity_set` and update them to use the correct public API.

Since we are not documenting the Conduce entity structure we can't include this function in our official API.